### PR TITLE
Remove old functions from assisted_deployment.sh and print net-dhcp-leases

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: scripts/assisted_deployment.sh [hunk 1]
```diff
@@ -1,20 +1,21 @@
 #!/usr/bin/env bash
 
-function destroy_all() {
```
Comment (tsorya) [inline scripts/assisted_deployment.sh:3]: Why?

File: scripts/assisted_deployment.sh [hunk 2]
```diff
@@ -43,32 +45,4 @@ function update_vips() {
     virsh net-update test-infra-net-${NAMESPACE} add dns-host "<host ip='${INGRESS_VIP}'><hostname>assisted-service-assisted-installer.apps.${CLUSTER_NAME}-${NAMESPACE}.${BASE_DOMAIN}</hostname></host>"
 }
 
-# Delete after pushing fix to dev-scripts
-function wait_for_cluster() {
-    echo "Nothing to do"
-}
-
-#TODO ADD ALL RELEVANT OS ENVS
-function run() {
```
Comment (tsorya) [inline scripts/assisted_deployment.sh:52]: Why?

File: scripts/assisted_deployment.sh (hunk 1 above)
Comment (YuviGold) [reply scripts/assisted_deployment.sh:3]: it isn't used

File: scripts/assisted_deployment.sh (hunk 2 above)
Comment (YuviGold) [reply scripts/assisted_deployment.sh:52]: it isn't used

**Comment by YuviGold** [comment]
Not relevant anymore.

**Pull request closed without merging**